ext/dav1d.cmd

```diff
@@ -18,6 +18,8 @@ cd build
 : # macOS might require: -Dc_args=-fno-stack-check
 : # Build with asan: -Db_sanitize=address
 : # Build with ubsan: -Db_sanitize=undefined
-meson --default-library=static --buildtype release ..
-ninja
+: # Tell MSYS2 don't mess with local prefix, no-op for other platforms
+: ; export MSYS2_ARG_CONV_EXCL="--prefix="
+meson --default-library=static --buildtype release --prefix=/usr ..
```

Comment:
I'm changing dav1d to 'install', because otherwise header files are scattered everywhere and hard to specify using one include dir.